### DISCUSSION TITLE
[Feature] Automatic image deletion of already backed up photos from the phone

### DISCUSSION THREAD

**Discussion by willabytes** [immich-app/immich#12211] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

One feature I would really like to see is that once the photos are automatically backed up from the phone's internal storage, they get deleted from it, making it so you don't have to delete them on your phone manually. Maybe also a setting that sets how long after it was taken it gets deleted from the phone. Like, if you take a photo, it gets backed up, and then after for example 5 days it gets deleted from the internal storage so you have it locally for a few days.

Just let me know if I should clarify anything! :)
AMAZING work by the way.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#165

**Reply by willabytes** [reply]
Sorry! Should've searched harder after a duplicate...
